python/lsst/pipe/tasks/ingestCalibs.py

```diff
@@ -38,6 +39,8 @@ def getCalibType(self, filename):
             obstype = "fringe"
         elif "sky" in obstype:
             obstype = "sky"
+        elif "defects" in obstype:
+            obstype = "defects"
```

Comment:
This is getting a bit long and the stutter worries me (a typo could easily creep in). Consider a loop, such as:
```
remap_dict = {"zero": "bias"}
for name in ("flat", "zero", "bias", "dark", "fringe", "sky", "defects"):
    if name in obstype:
        return remap_dict.get(name, name)
```